native_shortuuid/templates/admin/native_shortuuid_search_form.html

```diff
@@ -0,0 +1,21 @@
+{# this template is a copy of with small modification "admin/search_form.html" #}
+{% load i18n static %}
+{# if cl.search_fields #}
```

Comment:
Why do we want to show search widget although we don't have search_fields?

Reply:
Because we also have `NativeUUIDSearchMixin.search_uuid_fields`,
and if the model doesn't have any uuids it don't make since to have it inherit from the `NativeUUIDSearchMixin` in the first place.

I'm doing this because on one of our systems we allow searching for UUIDs only which made the search _fields empty, and that resulted in now showing the search bar!